docs/guide/loan.md

```diff
@@ -0,0 +1,912 @@
+---
+description: Loan blockchain using Starport
+order: 6
+title: "Advanced Module: DeFi Loan"
+---
+
+# Loan Module
+
+As a rapidly growing industry in the blockchain ecosystem, (decentralized finance) DeFi is spurring innovation and revolution in spending, sending, locking, and loaning cryptocurrency tokens.
+
+One of the many goals of blockchain is to make financial instruments available to everyone. A loan in blockchain DeFi can be used in combination with lending, borrowing, spot trading, margin trading, and flash loans.
+
+With DeFi, end users can quickly and easily access loans without having to submit their passports or background checks like in the traditional banking system.
+
+In this tutorial, you learn about a basic loan system as you use Starport to build a loan module.
+
+**You will learn how to**
+
+* Scaffold a blockchain
+* Scaffold a Cosmos SDK loan module
+* Scaffold a list for loan objects
+* Create messages in the loan module to interact with the loan object
+* Interact with other Cosmos SDK modules
+* Use an escrow module account
+* Add application messages for a loan system
+  * Request loan
+  * Approve loan
+  * Repay loan
+  * Liquidate loan
+  * Cancel loan
+
+**Note:** The code in this tutorial is written specifically for this learning experience and is intended only for educational purposes. This tutorial code is not intended to be used in production.
+
+## Module Design
+
+A loan consists of:
+
+- An `id`
+- The `amount` that is being lent
+- A `fee` as cost for the loan
+- The borrowing party provides a `collateral` to request a loan 
+- A loan has a `deadline` for repayment, after which the loan can be liquidated
```

Comment:
In that context, I feel "due" is a softer term after which you might be given an extension subjected to some penalty. But since after the "deadline" we liquidate the collateral, I think its more appropriate to use "deadline"